torch/onnx/symbolic_opset9.py

```diff
@@ -2194,6 +2194,16 @@ def prim_shape(g, self):
     return g.op('Shape', self)
 
 
+def prim_dtype(g, self):
+    dtype = sym_help._try_get_scalar_type(self)
+    dtype = sym_help.cast_pytorch_to_scalar_type[dtype]
+    return g.op("Constant", value_t=torch.IntTensor([dtype]))
+
+
+def prim_tolist(g, input, dim_val, elem_ty_val):
```

Comment:
What is the use-case of dim_val and elem_ty_val?

Reply:
tolist creates a list out of a tensor, dim_val and elem_ty_val are part of prim::tolist signature. dim_val represent dimension of a tensor and elem_ty_val is the element type, they are not needed for export since everything in onnx is a tensor

Reply:
Not sure, cause we have sequence types in ONNX.
So does this op always get a tensor and return a lis?
Can we try export with SplitToSequence?

Reply:
to list is meant to be as close in implementation as python function tensor_to_list, which takes a tensor and returns a list. Also SplitToSequence is only available from opset 11

Reply:
It's okay. The symbolic needs to replicate the torch op behavior anyways. We need to make sure the symbolic for this op is implement to cover all cases. It's okay if you think it's not exportable to lower opsets.